core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -117,6 +117,14 @@ package object config {
     .bytesConf(ByteUnit.MiB)
     .createOptional
 
+  private[spark] val DRIVER_MIN_MEMORY_OVERHEAD = ConfigBuilder("spark.driver.minMemoryOverhead")
+    .doc("The minimum amount of non-heap memory to be allocated per driver in cluster mode, " +
+      "in MiB unless otherwise specified. Overrides the default value of 384Mib." +
+      " This value is ignored if spark.driver.memoryOverhead is set directly.")
+    .version("3.5.2")
```

Comment:
this should be 4.0.0. This is a new feature so generally wouldn't be back ported, at least without discussion.